README.md

```diff
@@ -43,38 +43,40 @@ the central point of interaction for users:
   to make sure you have the latest versions. 
 
 - When hitting the "sendToChat/sendToChat/forward" button on any app for
-  the first time a download will be triggered (using the send/receive message webxdc APIs). 
+  the first time a download will be triggered (using the send/receive message WebXDC APIs).
 
 - For now, any message you send to the store bot will trigger it to send 
   the current store xdc in a new message. Later on we rather want to use 
   an update mechanism so there will only need to be a single store xdc app in
   the store bot chat. 
 
-## Setting Up the Bot
+## Setting up the bot
 
-### Requirements
+### Installing pre-built binary releases
 
-You need to have `rust` and `node` installed.
-Optionally, have [pnpm](https://pnpm.io/) installed.
+TODO: Download the latest release from https://github.com/webxdc/store/releases,
+extract the compressed archive to the desired location, it containts the bot command line
+program and required assets.
 
-### Setting up the bot 
+To check the program is installed correctly run:
 
-Go to the `frontend` folder and install the packages using `pnpm install` or `pnpm i`.
-If you do not have `pnpm` installed, you can run `npx pnpm install` to use `pnpm` 
-without installing it globally or use `npm install` instead.
-However, if you use `npm install`, the latest versions of dependencies will be installed
-instead of the ones listed in the `frontend/pnpm-lock.yaml` file.
+```
+/path/to/xdcstore version
+```
 
-Build the frontend by running `npm run build` or `pnpm run build`.
-This step creates a `bot-data` directory in the root of the repository
-with files `store.xdc`, `review-helpr.xdc` and `submit-helper.xdc`.
+You should see the version code printed on the screen.
+
+If you want to install from source code, read:
+https://github.com/webxdc/store#developing--building-from-source
+
+### Running the bot
 
 To run the bot, set the environment variables
 `addr` and `mail_pw` with the login and password
 and use the command `start`:
 
 ```
-    addr=bot@example.org mail_pw=My_P4ssword cargo run -- start
+    addr=bot@example.org mail_pw=My_P4ssword /path/to/xdcstore start
```

Comment:
overall i don't like this `/path/to/xdcstore` especially since above there is no example unpacking which could then be used here with a relative path.